src/components/[guild]/EditGuild/components/Events/EventInput.tsx

```diff
@@ -29,34 +31,46 @@ const logos: Record<EventSourcesKey, string> = {
 }
 
 const EventInput = ({ eventSource }: Props) => {
-  const { register, setValue } = useFormContext<GuildFormType>()
+  const {
+    register,
+    setValue,
+    formState: { errors },
+  } = useFormContext<GuildFormType>()
+
   const link = useWatch({
     name: `eventSources.${eventSource}`,
   })
 
   if (!link && link !== "") return null
 
   return (
-    <InputGroup size={"lg"}>
-      <InputLeftElement>
-        <Circle bgColor={"gray.900"} size={5}>
-          <Img boxSize={5} src={logos[eventSource]} borderRadius={"full"} />
-        </Circle>
-      </InputLeftElement>
-      <Input
-        {...register(`eventSources.${eventSource}`)}
-        size={"lg"}
-        placeholder={placeholders[eventSource]}
-      />
-      <InputRightElement>
-        <CloseButton
-          aria-label="Remove link"
-          size="sm"
-          rounded="full"
-          onClick={() => setValue(`eventSources.${eventSource}`, undefined)}
+    <FormControl isInvalid={!!errors?.eventSources?.[eventSource]} isRequired>
+      <InputGroup size={"lg"}>
+        <InputLeftElement>
+          <Circle bgColor={"gray.900"} size={5}>
```

Comment:
I think we'll always have an event source image (right?), so we don't need this wrapper circle here

Reply:
Yes, but the link3 logo looks misleading if there is no background.

![image](https://github.com/agoraxyz/guild.xyz/assets/139131231/8e7bda5b-9b0b-4bd3-b432-c847aa0c5337)